sdk/resourcehealth/Azure.ResourceManager.ResourceHealth/src/autorest.md

```diff
@@ -0,0 +1,60 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+azure-arm: true
+csharp: true
+library-name: ResourceHealth
+namespace: Azure.ResourceManager.ResourceHealth
```

Comment:
Is this namespace approved by arch board?

Reply:
This is old service which already have Track 1 sdk